### PR TITLE
Update confirm run peribolos and readme file

### PR DESCRIPTION
**Pull request by huiwangredhat**
Update confirm value to run peribolos and readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+[Generating a user access token for a GitHub App](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-user-access-token-for-a-github-app).
+
+# Prerequisites
+- [Registered a github app](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app) and record the client ID
+- [Installed the app](https://docs.github.com/en/apps/using-github-apps/installing-your-own-github-app) with reasonable permission
+- [Enable the 'device flow'](https://docs.github.com/en/apps/creating-github-apps/writing-code-for-a-github-app/building-a-cli-with-a-github-app#about-device-flow-and-user-access-tokens) of the app
+- [Generate a github app user access token](https://docs.github.com/en/apps/creating-github-apps/writing-code-for-a-github-app/building-a-cli-with-a-github-app#write-the-cli)
+
+# References
+- [Peripolos CLI](https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/)
```
Comment (jpower432) [inline README.md:18]:
```suggestion
- [Peribolos CLI](https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/)
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+- [Peripolos source](https://github.com/kubernetes-sigs/prow/tree/main/cmd/peribolos)
```
Comment (jpower432) [inline README.md:19]:
```suggestion
- [Peribolos source](https://github.com/kubernetes-sigs/prow/tree/main/cmd/peribolos)
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
-This repository will apply peribolos to manage orgnization complytime.
+This repository will apply peribolos to manage organization complytime.
 
-#References
-[Peripolos CLI](https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/)
-[Peripolos source](https://github.com/kubernetes-sigs/prow/tree/main/cmd/peribolos)
+To use Peribolos to manage organization, the base requirement is Go setup.
+When running Peribolos, it needs permission to access the organization,
+and repository resources. The github app installation access token has no
```
Comment (jpower432) [inline README.md:5]:
```suggestion
and repository resources. The GitHub App installation access token has no
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+permission for endpoint user when peribolos updates members of organization. 
+The github app user access token could help. A user access token only has
```
Comment (jpower432) [inline README.md:7]:
```suggestion
The GitHub App user access token could help. A user access token only has
```

**Review by jpower432 — COMMENTED** [review]
This looks good! Minor comments around wording/typos (mostly nits).

File: README.md (hunk 1 above)
Comment (huiwangredhat) [reply README.md]: Thank you. Updated.

File: README.md (hunk 2 above)
Comment (huiwangredhat) [reply README.md]: Thanks :). Updated.

File: README.md (hunk 3 above)
Comment (huiwangredhat) [reply README.md]: Yes, it was updated.

File: README.md (hunk 4 above)
Comment (huiwangredhat) [reply README.md]: Thanks, updated.

**Review by jpower432 — APPROVED** [review]
LGTM

**Review by jpower432 — APPROVED** [review]
LGTM

**Pull request merged**